doc/source/userguide.rst

```diff
@@ -120,7 +120,15 @@ size:
 3-D
 ---
 
-...
+For an input array of shape $(B, C, N_1, N_2, N_3)$, a Solid Harmonic Scattering with $J$
+scales and $L$ angular frequencies, which applies $P$ different types of $\mathcal L_p$ 
+spatial averaging will result in an output of shape
+
+$(B, C, L, J + J(J - 1) / 2, P).$
+
```

Comment:
$(B, C, L, J + J(J - 1) / 2, P).$
 <->
$(B, C, L, J + J(J - 1) / 2, P)\,.$

Reply:
`.$` to `,.$`?

Reply:
\, sorry, just an insecable space between a point and the paranthesis

Reply:
So that's `~.$` instead of `.$`?

Reply:
(Called a [non-breaking space](https://en.wikipedia.org/wiki/Non-breaking_space).)

Reply:
damned, github parses my stuff, yes "vague" or "backslash virgule" is ok for me

Reply:
yeah a https://fr.wikipedia.org/wiki/Espace_ins%C3%A9cable

Reply:
but this is English. Non insecable spaces like that in English. I might still add one to make you happy though.

Reply:
ok sorry non non

Reply:
I fixed a bug and added an inescapable space

Reply:
i'm pros merging but maybe u don't like non-break space(i thought they should always surround equations)

Reply:
now I like it. Let's merge